gslib/commands/rsync.py

```diff
@@ -126,8 +126,8 @@
     gsutil rsync -r data gs://mybucket/data
 
   If you have a large number of objects to synchronize you might want to use the
-  gsutil -m option, to perform parallel (multi-threaded/multi-processing)
-  synchronization:
+  gsutil -m option (see "gsutil help options"), to perform parallel
```

Comment:
The CL shows this as a link, do we want to add the link here?

Reply:
We'll have to see how the generation process goes, but my understanding is there's logic in the generator that converts this into a link. If not, I'll definitely come back to this to make sure it does.

Reply:
Ah I see. Sounds good.